powerform/pf-deployment-context/classes/pf-build-context.ps1

```diff
@@ -0,0 +1,10 @@
+Class PfBuildContext {
```

Comment:
This is now the new class that will drive different "build sections" within the deployment context